keyboards/plexus75/info.json

```
@@ -0,0 +1,22 @@
+{
+    "keyboard_name": "Plexus75", 
+    "url": "", 
+    "maintainer": "npspears", 
+    "width": 15, 
+    "height": 5, 
+    "layouts": {
+	"default": {
```

Needs to be an exact match to the name used in the keyboard's header file (`plexus75.h`).

```suggestion
        "LAYOUT_2x2u": {
```